src/Symfony/Component/Validator/composer.json

```diff
@@ -53,6 +54,7 @@
         "symfony/config": "",
         "egulias/email-validator": "Strict (RFC compliant) email validation",
         "symfony/property-access": "For accessing properties within comparison constraints",
+        "symfony/property-info": "To automatically add NotNull and Type constraints",
```

Comment:
```patch
- To automatically add NotNull and Type constraints
+ To automatically add constraints based on properties metadata
```

?

Reply:
It is interesting to tell which one (because it may be confusing with the Doctrine loader)

Reply:
My worry is that it will because out of date at some point and we won't think about changing it :)

Reply:
@teohhanhui it is, because you don't need to install the same dependencies depending the constraints you want to populate.

Reply:
"To automatically add constraints based on NotNull and Type annotations"